index.html

```diff
@@ -2031,6 +2047,91 @@ <h2>
         </dd>
       </dl>
     </section>
+    <section data-dfn-for="AddressParts">
+      <h2>
+        <dfn>AddressParts</dfn> enum
+      </h2>
+      <pre class="idl">
+        enum AddressParts {
+          "addressLine",
+          "city",
+          "country",
+          "dependentLocality",
+          "organization",
+          "phone",
+          "postalCode",
+          "recipient",
+          "region",
+          "sortingCode"
+        };
+      </pre>
+      <p>
+        The {{AddressParts}} enum is used to represent individual parts of a
+        <a>physical address</a>.
+      </p>
+      <dl>
```

Comment:
I have a suspicion that some of these might be linking to the IDL definitions instead of the concepts... if they are: 

```suggestion
      <dl data-link-for="">
```

Reply:
I checked and they all link to Section 12 Physical Address. I think this is what we expect, right?

Reply:
Ok, cool. As long as they are fine, then no problem :)